packages/shared/lib/core/profile-manager/actions/events-handlers/handleTransactionInclusionEvent.ts

```diff
@@ -44,23 +41,37 @@ export function handleTransactionInclusionEventInternal(
     }
 
     if (activity?.type === ActivityType.Governance) {
-        if (inclusionState === InclusionState.Confirmed) {
-            updateSelectedAccount({ isTransferring: false })
-            closePopup(true)
+        handleGovernanceTransactionInclusionEvent(accountIndex, inclusionState, activity)
+    }
+
+    updateClaimingTransactionInclusion(transactionId, inclusionState, accountIndex)
+}
 
-            if (get(hasToRevote)) {
-                openPopup({
-                    id: PopupId.Revote,
-                    preventClose: true,
-                    hideClose: true,
-                })
+function handleGovernanceTransactionInclusionEvent(
+    accountIndex: number,
+    inclusionState: InclusionState,
+    activity: GovernanceActivity
+): void {
+    if (inclusionState === InclusionState.Confirmed) {
+        // TODO: Normally we update active account after a wallet.rs returns a transaction
+        // With governance we wait for the transaction confirmation
+        // we should think about making this consistent in the future
+        updateActiveAccount(accountIndex, { isTransferring: false })
+        // TODO: move this
+        closePopup(true)
```

Comment:
move where? 
and can this be addressed in this pr?

Reply:
It would be part of a larger refactor that would change the way the popup is closed. It's out of scope for this PR.